### PR TITLE
Add borderless table feature wrt #22638

### LINKED ISSUE #22638: Table borderless
Hello 
Would be interesting include some CSS to format table without borders (borderless)

```css
.table-borderless > tbody > tr > td,
.table-borderless > tbody > tr > th,
.table-borderless > tfoot > tr > td,
.table-borderless > tfoot > tr > th,
.table-borderless > thead > tr > td,
.table-borderless > thead > tr > th {
    border: none;
}
```

Best regards
Wanderlei

### PR DESCRIPTION
**Pull request by Varunram**
Fixes #22638.

CC @mdo @Johann-S

### CODE REVIEW

File: scss/_tables.scss [hunk 1]
```diff
@@ -148,6 +148,24 @@
 }
 
 
+.table-borderless {
+  tbody tr {
+    td, th {
+      border: none;
```
Comment (Varunram) [inline scss/_tables.scss]: Thanks Hound bot!

**Comment by mdo** [comment]
This doesn't seem like the right solution in the code. I haven't tested this, but it could probably all be reduced to something like maybe?

```scss
.table-borderless {
  td, th {
    border: 0;
  }
}
```

All we need to do is override the default `.table` styles, so perhaps compare against that opening `.table` ruleset?

**Comment by Varunram** [comment]
I guess it should work without the explicit `tbody`s. Thanks for the time @mdo, Updating my PR

**Comment by mdo** [comment]
_Making myself a note that this needs docs for when we're ready for it._

**Comment by XhmikosR** [comment]
@Varunram: can you add the new class in our docs?

**Comment by Varunram** [comment]
Sure, let me rebase the PR on master and revert all redundant commits.

**Comment by XhmikosR** [comment]
@mdo: LGTY?

**Comment by sanspace** [comment]
@mdo @cvrebert Is this to be added in a future release?

**Pull request merged**